bioblend/cloudman/__init__.py

```diff
@@ -365,10 +365,17 @@ def __init__(self, url, password, **kwargs):
         else:
             super().__init__(None, None)
         self.config = kwargs.pop('cloudman_config', None)
+        self.use_ssl = False
         if not self.config:
             self.password = password
+            self.verify = kwargs.get("verify",None)
+            self.use_ssl = kwargs.get("use_ssl",kwargs.get("verify",None) is not None)
         else:
             self.password = self.config.password
+            if self.config.kwargs.get('use_ssl',False):
+                self.verify = kwargs.get("verify",False)
```

Comment:
Should this be
```suggestion
                self.verify = self.config.kwargs.get("verify", False)
```

Reply:
There is also the issue that depending on whether this is an instance coming via the launcher or not, we may not have access to the config to determine whether this is a https url or not. I'm not deigning to introduce new user-configuration parameters (though y'all can if you like). In this case, if the user has asked for ssl in the config, default is False (no verify, needed for self signed certificates), but the init arguments, if present can override. Without the launch config, the default is None. So no, this code is as I intend.

Reply:
OK, I see what you mean (I think). I've pushed a commit to allow both `use_ssl` and `verify` to be set in the config, but overridable in `CloudManInstance.__init__()` kwargs. Does it make sense and work for you?